linux/keyman-config/keyman_config/install_kmp.py

```diff
@@ -246,7 +246,25 @@ def install_kmp_user(inputfile, online=False, language=None):
         raise InstallError(InstallStatus.Abort, message)
 
 
+def _normalize_language(supportedLanguages, language):
```

Comment:
This is probably a bit too straightforward. We should be normalizing according to langtags.json. I am scheduled to do that work for Windows in a couple of sprints and was hoping to define the algorithm then.

Reply:
I agree, this is a hack, not a real solution. However, it's better than what we currently have, so I suggest to merge this and then implement the better solution once you figured out the algorithm and someone has time to work on it.

Reply:
Ok. Can you document accordingly in the code with a TODO?